content/_guides/sasl/emacs-erc.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Configuring SASL for Emacs ERC
+category: sasl
+credits: androclus
+---
+
+[ERC](https://www.emacswiki.org/emacs/ERC) is an IRC client which comes as a package for Emacs.
+
+## A. Set up ERC IRC client
+
+If you have ERC already installed and running on your emacs system, then skip to part B.
+Otherwise, set up your ERC with basic functionality, as follows:
+
+1. Download and Install the erc package from melpa, etc. (*M-x package-list-packages*)
+2. Set up your ERC for plain (unregistered) sign-in.
+  - There are some basic setup instructions [here](https://www.emacswiki.org/emacs/ERC)
+  - Below is my own setup (before adding the SASL stuff below), though by no means do you need all this to set up a basic ERC for yourself.
+```
+;; ======== ERC START ==============================
+
+;; Most of these are just for convenience options. I believe the only
+;; absolutely necessary one here is the my-erc-connect() function I
+;; define for connecting.
+
+;; for auto-joining networks and channels when starting up ERC
+(require 'erc-join)
+(erc-autojoin-enable)
+
+;; Logging of conversations (mostly for when I forget to save peoples’
+;; answers when I ask a programming question).
+
+;; turn on logging
+(setq erc-log-mode t)
+;; write to log file when new text is added to a logged ERC buffer.
+(setq erc-log-write-after-insert t)
+;; write to log file after every message I send.
+(setq erc-log-write-after-send t)
+;; Include network name in the buffers
+(setq erc-rename-buffers t)
+
+;; from local erc-log.el and tweeked
+;; see also https://www.emacswiki.org/emacs/ErcLogging
+(require 'erc-log)
+(files--ensure-directory "~/.config")
+(files--ensure-directory "~/.config/erc")
+(files--ensure-directory "~/.config/erc/logs")
+(setq erc-log-channels-directory "~/.config/erc/logs/")
+(erc-log-enable)
+
+;; don't ask me for my nickserv password
+(setq erc-prompt-for-nickserv-password nil)
+
+;; A list of channels to auto-join
+(setq erc-autojoin-channels-alist
+	'(("irc.libera.chat" "#emacs" "#guile" "#scheme" "#erc" "#godot")))
+;; OR, if NONE..
+;; (setq erc-autojoin-channels-alist nil)
+
+;; from http://www.john2x.com/emacs.html
+;; Set the prompt to use the channel name.
+(setq erc-prompt  (lambda () (concat (buffer-name) "> ")))
+
+;; beautify the text on screen
+(setq erc-fill-function 'erc-fill-static)
+(setq erc-fill-static-center 10)
+
+;; my main function for connecting to libera.chat
+;; (see also https://libera.chat/guides/connect for regional hostnames)
+(defun my-erc-connect ()
+  (interactive)
+  (erc :server "irc.libera.chat" :port 6667 :nick "MYNICK" :full-name "MY NAME"))
```

Comment:
We shouldn't recommend to people to connect to port `6667`. Does ERC support TLS?

Reply:
Yes, ERC supports TLS. And I can change the port to 6697, will that work?

I can make these two changes, and resubmit with a new pull request -- will that work for you?